recipes/crystmatch/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "crystmatch" %}
+{% set version = "1.1.2" %}
+{% set python_min = "3.9" %}
```

Comment:
You don't need to set this since it's already the global pinned version that conda-forge uses.

Reply:
Thank you for your kind advice! I have removed these lines now.